config/jobs/kubernetes/sig-release/kubernetes-builds.yaml

```
@@ -164,3 +164,34 @@ periodics:
     testgrid-alert-email: release-managers+alerts@kubernetes.io
     testgrid-dashboards: sig-release-releng-informing
     testgrid-tab-name: build-packages-rpms
+
+- interval: 6h
+  name: ci-fast-forward
+  cluster: k8s-infra-prow-build-trusted
```

```suggestion
  cluster: k8s-infra-prow-build
```

I think you missed my updated comment. Sorry for the confusion. :-D

---

No worries, fixed as suggested :+1: